x-pack/plugins/alerting/server/task_runner/rule_action_helper.ts

```diff
@@ -69,6 +91,10 @@ export const isSummaryActionThrottled = ({
 };
 
 export const generateActionHash = (action?: RuleAction) => {
+  if (action != null && isSystemAction(action)) {
+    return `system-action:${action?.actionTypeId || 'no-action-type-id'}:summary`;
```

Comment:
**nit:** No need for `action?.actionTypeId || 'no-action-type-id'` as it can be `action.actionTypeId`.